gsad/src/gsad_gmp.c

```diff
@@ -2014,12 +1598,12 @@ get_many (gvm_connection_t *connection, const char *type,
           else
             filt_id = FILT_ID_USER_SETTING;
         }
-      else if ((strcmp (filter, "sort=nvt") == 0)
-               && ((strcmp (type, "note") == 0)
-                   || (strcmp (type, "override") == 0)))
+      else if (str_equal (filter, "sort=nvt")
+               && (str_equal (type, "note")
+                   || str_equal (type, "override") == 0))
         filt_id = FILT_ID_USER_SETTING;
-      else if ((strcmp (filter, "apply_overrides=1") == 0)
-               && (strcmp (type, "task") == 0))
+      else if (str_equal (filter, "apply_overrides=1")
+               && str_equal (type, "task") == 0)
```

Comment:
Same here

Reply:
All fixed now. Btw. I am not sure if all conditions in this path are correct. The code is confusing to me